### PR TITLE
chore: refactor ops in command line

### PR DESCRIPTION
**Pull request by tokers**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/cli/ops.lua [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    end
+
+    return true
+end
+
+
+local function get_openresty_version()
+    local str = "nginx version: openresty/"
+    local ret = util.execute_cmd("openresty -v 2>&1")
+    local pos = str_find(ret, str)
```
Comment (spacewander) [inline apisix/cli/ops.lua:84]: Would be better to use `str_find(ret, str, 1, true)`

File: apisix/cli/ops.lua [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+    if pos then
+        return str_sub(ret, pos + #str)
+    end
+
+    str = "nginx version: nginx/"
+    pos = str_find(ret, str)
```
Comment (spacewander) [inline apisix/cli/ops.lua:90]: Ditto

File: apisix/cli/ops.lua (hunk 1 above)
Comment (tokers) [reply apisix/cli/ops.lua]: @spacewander Fixed.

File: apisix/cli/ops.lua (hunk 2 above)
Comment (tokers) [reply apisix/cli/ops.lua]: @spacewander Fixed.

**Review by spacewander — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**